dev/tools/gen_keycodes/data/keyboard_key.tmpl

```diff
@@ -161,7 +161,7 @@ class LogicalKeyboardKey extends KeyboardKey {
   /// “o” for [keyLabel], but would return “ö” for [RawKeyEvent.character].
   ///
   /// {@macro flutter.services.RawKeyEventData.keyLabel}
-  final String keyLabel;
+  final String? keyLabel;
```

Comment:
/cc @dkwingsmt or @gspencergoog Is it expected that the label can be null? If so, can we add something to the docs here to explain what that means?

Reply:
null seems to be possible regarding https://github.com/flutter/flutter/blob/7ec2d360bce51f010554be8d80572af4748a4a95/dev/tools/gen_keycodes/lib/keyboard_keys_code_gen.dart#L61-L73

Reply:
Yes, null is possible, since it's possible that we don't have the key label data for a key.

Reply:
Can we update the doc comment here to mention that?

Reply:
Done